packages/client/hmi-client/src/components/models/tera-stratified-model-configuration.vue

```diff
@@ -186,6 +188,9 @@ const vFocus = {
 function updateModelConfigName(configIndex: number) {
 	modelConfigurations.value[configIndex].name = modelConfigInputValue.value;
 	updateModelConfiguration(modelConfigurations.value[configIndex]);
+	setTimeout(() => {
+		emit('update-model-configuration');
+	}, 800);
```

Comment:
Could this be triggered by waiting from a response of `updateModelConfiguration()` instead of a timeout?

Reply:
Not yet, need to wait for TDS change for ElasticSearch.